### DISCUSSION TITLE
Get title column method

### DISCUSSION THREAD

**Discussion by ulissesprev** [l-lin/angular-datatables#1609] [General]
Hi, can you give me a hand on how I can get the column title for future comparison?

I have this code but it doesn't work on angular, what can I be wrong?

![image](https://user-images.githubusercontent.com/34987079/137762825-bc2b834d-b88e-4a03-8cf9-e823a1b8a421.png)

I have in jquery codes working that way.

thank you

**Comment by shanmukhateja** [comment]
What happens if you try it this way:
```
switch (column['title']) {
```

This should make the error go away.

**Reply by ulissesprev** [reply]
Hello
No error occurs, but is returned undefined

![image](https://user-images.githubusercontent.com/34987079/137944471-1d6a5073-c39c-4a2f-a0dc-b74277058a5d.png)

![image](https://user-images.githubusercontent.com/34987079/137943968-4a563028-c3cc-4feb-8517-1d47a7c942b3.png)

**Reply by shanmukhateja** [reply]
Hi,

Please provide a reproducible repo or StackBlitz to investigate.

**Reply by ulissesprev** [reply]
Hi, 
That line of code gave me back what I needed. Thanks.

`column.header().textContent`

**Comment by ulissesprev** [comment]
I got something using this line of code, but I don't know if it's safe to use...

`column.header().textContent`
